index.html

```diff
@@ -2276,12 +2178,37 @@ <h2>
 method-specific <a>DID scheme</a> define as few <code>method-specific-id</code>
 formats as possible.
       </p>
+
+      <p>
+The authors of a new <a>DID method</a> specification SHOULD use a method name
```

Comment:
why is this not a MUST?

Reply:
... because people objected to a MUST in the CG. The argument was "How can it be a MUST when there is no official registry for DID Methods!?"... now that the group has centralized registration, I expect it could be changed to a MUST. 

Also, as an editor, I chose NOT to change any normative content and keep this purely editorial to avoid objections from the group when merging. 

Unfortunately, you're missing a lot of context from the F2F meeting... much of what you're raising came up during the meeting during hallways discussion and topic discussion. We will get to it in time, but not in this PR.